vassal-app/src/main/java/VASSAL/counters/KeyBuffer.java

```diff
@@ -109,6 +121,18 @@ public Command keyCommand(javax.swing.KeyStroke stroke) {
     for (GamePiece p : targets) {
       bounds.addPiece(p);
       p.setProperty(Properties.SNAPSHOT, ((PropertyExporter) p).getProperties()); // save state prior to command
+      
+      // Send most recent click point location
+      GamePiece bp = Decorator.getInnermost(p);
+      if (bp instanceof BasicPiece) {
+        Command c3 = new NullCommand();
+        c3 = ((BasicPiece)bp).setPersistentProperty(BasicPiece.CLICKED_X, String.valueOf(clickPoint.x));
+        c3.append (((BasicPiece)bp).setPersistentProperty(BasicPiece.CLICKED_Y, String.valueOf(clickPoint.y)));
+        if ((comm != null) && !comm.isNull()) {
+          comm = comm.append(c3);
+        }
+      }              
+      
       Command c2 = p.keyEvent(stroke);
       comm = comm.append(c2);
```

Comment:
This can be replace by

comm = comm.append(p.keyEvent(stroke);

Reply:
done